gulpfile.babel.js

```diff
@@ -25,17 +25,30 @@ if (process.env.DEBUG) {
   defaultArgs.unshift("--debug");
 }
 
-gulp.task("hugo", (cb) => buildSite(cb));
-gulp.task("hugo-preview", (cb) => buildSite(cb, ["--buildDrafts", "--buildFuture"]));
-gulp.task("build", ["css", "js", "hugo"]);
-gulp.task("build-preview", ["css", "js", "hugo-preview"]);
+gulp.task("hugo", (cb) => {
+  let baseUrl = process.env.NODE_ENV === 'production' ? process.env.URL : process.env.DEPLOY_PRIME_URL;
```

Comment:
We don't want a hardcoded dependency on Netlify, we'll want to handle this differently.